client/elife-theme/src/elements/TextField.js

```diff
@@ -0,0 +1,44 @@
+import { css } from 'styled-components'
+import { th } from '@pubsweet/ui-toolkit'
+
+/**
+ * TODO
+ * this should be removed once it's implemented in pubsweet
+ * right now they don't have colorWarning
```

Comment:
`colorWarning` is back in [their TextField](https://gitlab.coko.foundation/pubsweet/pubsweet/blob/master/packages/ui/src/atoms/TextField.js#L23) - from [this MR](https://gitlab.coko.foundation/pubsweet/pubsweet/merge_requests/296/diffs#dbe470512771627c07b2f1fdb1d1ade7d82834d3)

Reply:
Have we changed this to colourWarning (UK spelling) now?

Reply:
Ah, nevermind. We're using colorWarning (misread @jsms90 edit on the other PR)!

Reply:
Cool, thanks @jsms90. I wouldn't worry too much about this now since we're supposed to merge this one today. That MR you linked to doesn't have everything I need so I need to do another MR to pubsweet myself. Also, I think we will end up renaming that `borderColor` function.

Reply:
How do you mean, it doesn't have everything you need? This exact function exists in pubsweet's `TextField`. So can we not remove the comment & the `borderColor` function altogether? :confused:

Reply:
Oh yeah, this function was included, my bad, thanks.